### ISSUE TITLE
Question: How to get space before/after colon in range based for-loop

### ISSUE THREAD

**Issue by arneforlie** [uncrustify/uncrustify#1839]
Is it possible to configure uncrustify to insert space before/after the colon in a range based for-loop?

For instance, I have the following test case:
```
#include <vector>

void
f()
{
	std::vector<int> v;
	int sum = 0;
	for (auto& e:v)
	{
		sum += e;
	}
}
```
where I want the for-loop to be formatted as:
```
	for (auto& e : v)
```
Is that possible?

I have looked at all the sp_* options but none of them seem to apply to the range based for-loop.

Uncrustify-0.67-128-03efec8

**Comment by CDanU** [comment]
```sql
# Control the spacing after ':' in 'for (TYPE VAR : EXPR)'. Only JAVA.
sp_after_for_colon              = add      # ignore/add/remove/force

# Control the spacing before ':' in 'for (TYPE VAR : EXPR)'. Only JAVA.
sp_before_for_colon             = add      # ignore/add/remove/force
```

Only Java ... well that needs a description update.

**Comment by CDanU** [comment]
Btw. you can try out single options faster with: https://github.com/CDanU/uncrustify_config

**Comment by arneforlie** [comment]
Will keep it mind for the next time I need to twiddle the options.

**Comment by arneforlie** [comment]
Btw: if you are going to modify the description, you may consider to add the words "range based for-loop" and perhaps not use All Caps because it leads many C++ programmers to think of MACROS.

**Comment by mwoehlke-kitware** [comment]
(FYI: The description is fixed in #1852.)

**Comment by mwoehlke-kitware** [comment]
I guess this is fixed now?
